vizro-core/examples/_dev/app.py

```diff
@@ -3,77 +3,36 @@
 import vizro.models as vm
 import vizro.plotly.express as px
 from vizro import Vizro
+from vizro.models._components.form._text_area import TextArea
+from vizro.models._components.form._user_input import UserInput
 
-gapminder = px.data.gapminder()
+iris = px.data.iris()
 
-child_container_one = vm.Container(
-    title="Container I",
-    layout=vm.Layout(grid=[[0, 1], [2, 2]]),
-    components=[
-        vm.Graph(
-            figure=px.line(
-                gapminder,
-                title="Graph 1 - Container I",
-                x="year",
-                y="lifeExp",
-                color="continent",
-            ),
-        ),
-        vm.Graph(
-            figure=px.scatter(
-                gapminder,
-                title="Graph 2 - Container I",
-                x="gdpPercap",
-                y="lifeExp",
-                size="pop",
-                color="continent",
-            ),
-        ),
-        vm.Graph(
-            figure=px.box(
-                gapminder,
-                title="Graph 3 - Container I",
-                x="continent",
-                y="lifeExp",
-                color="continent",
-            ),
-        ),
-    ],
-)
-
-child_container_two = vm.Container(
-    title="Container II",
-    layout=vm.Layout(grid=[[0, 1]]),
-    components=[
-        vm.Graph(
-            figure=px.line(
-                gapminder,
-                title="Graph 4 - Container II",
-                x="year",
-                y="lifeExp",
-                color="continent",
-            ),
-        ),
-        vm.Graph(
-            figure=px.scatter(
-                gapminder,
-                title="Graph 5 - Container II",
-                x="gdpPercap",
-                y="lifeExp",
-                size="pop",
-                color="continent",
-            ),
-        ),
-    ],
-)
+# Only added to container.components directly for dev example
+vm.Container.add_type("components", UserInput)
+vm.Container.add_type("components", TextArea)
```

Comment:
Not related to this PR, but we've already mentioned it, but worth discussing whether we even need a `Form` model or if we just want to fall back on the `Container` and add a default action if there is a Button inside. Will be relevant for the model interactions that are yet to come @maxschulz-COL